packages/govern-core/contracts/pipelines/GovernQueue.sol

```diff
@@ -212,13 +212,32 @@ contract GovernQueue is IERC3000, AdaptativeERC165, IArbitrable, ACL {
         }
     }
 
-    function veto(bytes32 _containerHash, bytes memory _reason) auth(this.veto.selector) override public {
-        queue[_containerHash].checkAndSetState(
-            GovernQueueStateLib.State.Scheduled,
-            GovernQueueStateLib.State.Cancelled
-        );
+    function veto(
+        ERC3000Data.Container memory _container,
+        bytes memory _reason
+    ) auth(this.veto.selector) override public {
```

Comment:
We should definitely look into #54 so we can have all functions tidy with a certain style. :)